process/Release.md

```diff
@@ -1,20 +1,29 @@
 **Scenario:** You want to release a new version
 
-- Add issues to release on JIRA
+- Notify team of release start
+- Set release start date to today
```

Comment:
In "[JIRA Project > Releases]"?

That one's up to you, I feel like we should be -slightly- specific about directions, but not too too specific, considering that the technologies used in our process are very much still in flux.